web/src/gear_menu.js

```diff
@@ -59,7 +59,7 @@ The menu itself has the selector
 The items with the prefix of "hash:" are in-page
 links:
```

Comment:
Small side comment, but I think this comment is close to 10 years out of date and can probably be deleted? I think people can figure out what is in it by reading the code.